source/particle/generator/uniform_radial.cc

```diff
@@ -171,15 +171,15 @@ namespace aspect
                                    Patterns::Double (0,360),
                                    "Minimum longitude coordinate for the region of particles "
                                    "in degrees. Measured from the center position.");
-                prm.declare_entry ("Maximum longitude", "3.1415",
+                prm.declare_entry ("Maximum longitude", "360",
                                    Patterns::Double (0,360),
                                    "Maximum longitude coordinate for the region of particles "
                                    "in degrees. Measured from the center position.");
                 prm.declare_entry ("Minimum latitude", "0",
                                    Patterns::Double (0,180),
                                    "Minimum latitude coordinate for the region of particles "
                                    "in degrees. Measured from the center position.");
-                prm.declare_entry ("Maximum latitude", "3.1415",
+                prm.declare_entry ("Maximum latitude", "180",
                                    Patterns::Double (0,180),
                                    "Maximum latitude coordinate for the region of particles "
```

Comment:
Isn't the maximal latitude 90 degrees? I don't know how this is used, but a latitude of 180 degrees doesn't make much sense.

Reply:
It looks like the "maximum latitude" and "minimum latitude" are actually colatitude.
Does it make more sense to clarify that in the documentation, or to change things so that latitude is between -90 and 90?

Reply:
My preference is to keep things compatible, but to change the documentation to explain what it really is.